pages/04.Reference/08.network/03.known-issues/docs.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Known Issues'
+published: true
+date: '08-14-2019 10:51'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+### Idle TCP sessions being closed
+
+**Problem Statement:**\
+When a virtual machine establishes a TCP connection to a remote server, it uses a random TCP source port.
+In order for return traffic to be allowed to flow into a VM in OpenStack, a dynamic inbound security group rule must be created, allowing traffic to flow to this random TCP port.
+This dynamic rule will expire in 60 seconds. If the server was quiet for more than one minute, the dynamic inbound security group rule will be deleted and the return traffic from the remote server will be rejected.
+
+**Solutions:**\
+To avoid running into this issue there are 3 possible solutions
```

Comment:
Please mention 4th solution: If both TCP client and server are hosted in same OpenStack account, security group rule could be created to allow all traffic from other "security group". It is more or less same thing as "default" security group in OpenStack - allows all traffic from instances that have this "default" security group attached to them.